x/stablestake/spec/README.md

```diff
@@ -0,0 +1,29 @@
+# Stablestake Module
+
+## Contents
+
+1. **[Concepts](01_concepts.md)**
+2. **[Usage](02_usage.md)**
+3. **[Keeper](03_keeper.md)**
+4. **[Protobuf Definitions](04_protobuf_definitions.md)**
+5. **[Functions](05_functions.md)**
+
+## References
+
+Resources:
+
+- [Elys Network Documentation](https://docs.elys.network)
+- [Cosmos SDK Documentation](https://docs.cosmos.network)
+- [GitHub Repository for Elys Network](https://github.com/elys-network/elys)
+
+## Overview
+
+The `stablestake` module in the Elys Network is designed to manage stable staking functionalities by providing efficient borrowing and lending mechanisms, managing interest rates, and ensuring the stability of the staking process. This module enhances the staking experience and efficiency within the network, ensuring seamless borrowing, interest calculation, and effective debt management.
+
+## Key Features
+
+- **Borrowing and Lending Management**: Efficiently manage borrowing and lending operations.
```

Comment:
Is it possible to add mechanism for this ?

Reply:
@amityadav0 the liabilities management mechanism are managed by the leverage lp module and can be found within the leverage lp module doc

Reply:
okay, so how does lending and borrowing works in this module ?
are lp tokens from commitment module considered as collateral for borrowing USDC ?